package.json

```
@@ -7,9 +7,13 @@
     "dev": "vite",
     "build": "tsc -b && vite build",
     "test": "vitest --ui",
+    "test:report": "vitest run",
```

Думаю лучше изменить название с `test:report` на `test`, а `test` который был обозвать `test:watch`
```suggestion
    "test": "vitest run",
    "test:watch": "vitest --ui",
```

---

Переименовала и сделала тесты coverage по аналогии так же двух видов.